crates/pop-cli/src/commands/new/parachain.rs

```diff
@@ -43,12 +49,6 @@ pub struct NewParachainCommand {
 		default_value = "1u64 << 60"
 	)]
 	pub(crate) initial_endowment: Option<String>,
-	#[arg(
-		short = 'p',
-		long,
-		help = "Path for the parachain project, [default: current directory]"
-	)]
-	pub(crate) path: Option<PathBuf>,
```

Comment:
Why remove it?
This was introduced by @weezy20  here: https://github.com/r0gue-io/pop-cli/pull/100  to keep consistency with `pop new contracts` where we have the flag path:

Reply:
As Frank mentioned, the path arg was never used. I recommend we only add it back in if it becomes needed.